### PR TITLE
Implement ASSERT_LT_FELT hint

### LINKED ISSUE #182: ASSERT_LE_FELT/ASSERT_LT_FELT Hints
List:

- `ASSERT_LE_FELT`
- `ASSERT_LE_FELT_EXCLUDED_0`
- `ASSERT_LE_FELT_EXCLUDED_1`
- `ASSERT_LE_FELT_EXCLUDED_2`
- `ASSERT_LT_FELT`

### PR DESCRIPTION
**Pull request by fmoletta**
Depends on #240 & #233 (Contains changes from both, please wait until one of them is merged into main and switch base accordingly to merge into base branch)
Closes #182

### CODE REVIEW

File: pkg/hints/hint_utils/hint_utils.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"strings"
+
+	. "github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	"github.com/pkg/errors"
+)
+
+func GetConstantFromVarName(name string, constants *map[string]Felt) (Felt, error) {
+	if constants == nil {
+		return Felt{}, errors.Errorf("Caled GetConstantFromVarName with a nil constants map. Var Name: %s", name)
```
Comment (mmsc2) [inline pkg/hints/hint_utils/hint_utils.go]:
```suggestion
		return Felt{}, errors.Errorf("Called GetConstantFromVarName with a nil constants map. Var Name: %s", name)
```
typo

File: pkg/hints/math_hints.go [hunk 2]
```diff
@@ -151,3 +154,253 @@ func sqrt(ids IdsManager, vm *VirtualMachine) error {
...
+	if err != nil {
+		return err
+	}
+	b := bFelt.ToBigInt()
+	rangeCheckPtr, err := ids.GetRelocatable("range_check_ptr", vm)
+	if err != nil {
+		return err
+	}
+	// Hint Logic
+	cairoPrime, _ := new(big.Int).SetString(CAIRO_PRIME_HEX, 0)
```
Comment (mmsc2) [inline pkg/hints/math_hints.go]: why the base of `SetString` is 0? should not be 16 if converting an hex value?

Comment (fmoletta) [reply pkg/hints/math_hints.go]: `CAIRO_PRIME_HEX` starts with "0x", and `SetString` uses the prefix to determine the base if base is 0

File: pkg/hints/hint_utils/hint_utils.go (hunk 1 above)
Comment (fmoletta) [reply pkg/hints/hint_utils/hint_utils.go]: File removed after update

**Review by jrchatruc — APPROVED** [review]

**Review by Juan-M-V — APPROVED** [review]

**Pull request merged**